packages/sync-actions/src/product-actions.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable max-len */
 import forEach from 'lodash.foreach'
 import uniqWith from 'lodash.uniqwith'
+import intersection from 'lodash.intersection'
+import without from 'lodash.without'
```

Comment:
would avoid lodash, yep its clean but it had performance cost, there were old PR removed some lodash cause it were impacting performance, but tbh that would do the job for now 😉